res/css/views/rooms/_RoomRecoveryReminder.scss

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2018 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_RoomRecoveryReminder {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    background-color: #fff8e3;
+    padding: 20px;
+}
+
+.mx_RoomRecoveryReminder_header {
+    font-weight: bold;
+    margin-bottom: 1em;
+}
+
+.mx_RoomRecoveryReminder_body {
+    margin-bottom: 1em;
+}
+
+.mx_RoomRecoveryReminder_button {
+    @mixin mx_DialogButton;
+    margin: 0 10px;
+}
+
+.mx_RoomRecoveryReminder_button.mx_RoomRecoveryReminder_secondary {
+    // flip colours for the secondary ones
+    font-weight: 600;
+    border: 1px solid $accent-color ! important;
+    color: $accent-color;
+    background-color: $accent-fg-color;
```

Comment:
I think the button colours should be covered by `.mx_Dialog button` / `mx_Dialog_primary` - they might not have been ported to the mixin.

Reply:
I wasn't sure how to best handle this... The `.mx_Dialog` styles aren't naturally available in this part since it isn't a model dialog, so that class isn't set. I think I'll extract the colour flip to a new mixin.